apps/console/src/public/deployment.config.json

```diff
@@ -4,6 +4,8 @@
         "homeRealmId": ""
     },
     "accountApp": {
+        "clientID": "MY_ACCOUNT",
```

Comment:
Let's update the deployment.config.json.j2 and framework as well, since this file is for local development only.

Reply:
Fixed with 82d4ff507d55ec7ed6c0b57080b6e89963594857